x-pack/plugins/maps/public/connected_components/toolbar_overlay/set_view_control/set_view_control.tsx

```diff
@@ -42,8 +74,91 @@ export class SetViewControl extends Component<Props, State> {
     lat: 0,
     lon: 0,
     zoom: 0,
+    coord: DEFAULT_SET_VIEW_COORDINATE_SYSTEM,
+    mgrs: "",
+    utm: {}
   };
 
+  static convertLatLonToUTM(lat, lon) {
+    const utmCoord = converter.LLtoUTM(
+      lat,
+      lon
+    );
+
+    let eastwest = 'E';
+    if (utmCoord.easting < 0) {
+      eastwest = 'W';
+    }
+    let norwest = 'N';
+    if (utmCoord.northing < 0) {
+      norwest = 'S';
+    }
+
+    utmCoord.zoneLetter = isNaN(lat) ? '' : converter.UTMLetterDesignator(lat)
+    utmCoord.zone = `${utmCoord.zoneNumber}${utmCoord.zoneLetter}`
+    utmCoord.easting = Math.round(utmCoord.easting);
+    utmCoord.northing = Math.round(utmCoord.northing);
+    utmCoord.str = `${utmCoord.zoneNumber}${utmCoord.zoneLetter} ${utmCoord.easting}${eastwest} ${utmCoord.northing}${norwest}`
+
+    return utmCoord;
+  }
+
+  static convertLatLonToMGRS(lat, lon) {
+    const mgrsCoord = converter.LLtoMGRS(lat,lon, 5);
+    return mgrsCoord;
+  }
+
+  static getViewString(lat, lon, zoom) {
+    return `${lat},${lon},${zoom}`;
+  }
+
+  static convertMGRStoUSNG(mgrs){
+    let eastNorthSpace, squareIdEastSpace, gridZoneSquareIdSpace
+    for(let i = mgrs.length - 1; i > -1; i--){
+      // check if we have hit letters yet
+      if(isNaN(mgrs.substr(i,1))){
+          squareIdEastSpace = i + 1
+          break;
+      };
+    }
+    gridZoneSquareIdSpace = squareIdEastSpace - 2
+    let numPartLength = mgrs.substr(squareIdEastSpace).length / 2;
+    // add the number split space
+    eastNorthSpace = squareIdEastSpace + numPartLength;
+    let stringArray = mgrs.split("");
+    
+    stringArray.splice(eastNorthSpace, 0, " ");
+    stringArray.splice(squareIdEastSpace, 0, " ");
+    stringArray.splice(gridZoneSquareIdSpace, 0, " ");
+    
+    let rejoinedArray = stringArray.join("");
+    return rejoinedArray;
+}
+
+static convertMGRStoLL(mgrs){
```

Comment:
To match our usual conventions, consider making this and other static functions into named functions. 

e.g.
```suggestion
function convertMGRStoLL(mgrs){
```